@here/harp-mapview/lib/VisibleTileSet.ts

```diff
@@ -1313,9 +1313,6 @@ export class VisibleTileSet {
             const tileKey = DataSourceCache.getKeyForTile(tile);
             if (!retainedTiles.has(tileKey)) {
                 retainedTiles.add(tileKey);
-                // Prevent label rendering issues when the style set is changing. Prevent Text
-                // element rendering that depends on cleaned font catalog data.
-                tile.clearTextElements();
```

Comment:
Would it make sense to actually clear the whole tile at this point, tile.clear() as its going to be reloaded/created

Reply:
We want to avoid flickering, hence marking the tile as dirty and reloading it.

Reply:
ah right ,sorry, I read this the wrong way